app/models/rubygem.rb

```diff
@@ -132,7 +132,7 @@ def indexed_versions?
 
   def owned_by?(user)
     return false unless user
-    ownerships.exists?(user_id: user.id)
+    ownerships.any? { |o| o.user_id == user.id }
```

Comment:
:thinking: AFAIK `any?` with block would not result into `SELECT 1` query if not preloaded already:

```ruby
irb(main):002:0> Rubygem.take.ownerships.exists?(user_id: 1)
  Rubygem Load (0.8ms)  SELECT "rubygems".* FROM "rubygems" LIMIT $1  [["LIMIT", 1]]
  Ownership Exists? (1.0ms)  SELECT 1 AS one FROM "ownerships" WHERE "ownerships"."rubygem_id" = $1 AND "ownerships"."user_id" = $2 LIMIT $3  [["rubygem_id", 35], ["user_id", 1], ["LIMIT", 1]]
=> false

irb(main):003:0> Rubygem.take.ownerships.any? { |o| o.user_id == 1 }
  Rubygem Load (0.7ms)  SELECT "rubygems".* FROM "rubygems" LIMIT $1  [["LIMIT", 1]]
  Ownership Load (0.6ms)  SELECT "ownerships".* FROM "ownerships" WHERE "ownerships"."rubygem_id" = $1  [["rubygem_id", 35]]
=> false
```